packages/create-cloudflare/templates-experimental/astro/templates/js/wrangler.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compatibility_date": "<TBD>",
+  "compatibility_flags": [
+    "nodejs_compat_v2"
```

Comment:
drop _v2?

Reply:
mh... weird.... why is the v2 there? https://github.com/cloudflare/workers-sdk/pull/7565/files 😕

Reply:
oh, I've updated only the ts version back then! 🤦